src/sentry/api/endpoints/sentry_apps.py

```diff
@@ -62,6 +62,7 @@ def post(self, request, organization):
             'redirectUrl': request.json_body.get('redirectUrl'),
             'isAlertable': request.json_body.get('isAlertable'),
             'isInternal': request.json_body.get('isInternal'),
+            'verifyInstall': request.json_body.get('verifyInstall'),
```

Comment:
We need to add

```python
data['verify_install'] = data['verifyInstall']
```

down around line 85. Otherwise we're passing `verifyInstall` to the mediator, when it expects `verify_install`.